plugins/modules/ec2_ami.py

```diff
@@ -339,6 +339,11 @@
     returned: when AMI is created or already exists
     type: str
     sample: "nat-server"
+enhanced_networking:
+    description: Specifies whether enhanced networking with ENA is enabled.
+    returned: when AMI is created or already exists
```

Comment:
I guess we should include version_added for all of them.

Reply:
Generally new things in "returns" weren't added by a specific version of the module, they started to appear with an API update (or sometimes a botocore update).  Since this PR doesn't itself update the return values I wouldn't bother.